### PR TITLE
Fix: Fix layout issues on the chapters page

### LINKED ISSUE #63: Fix layout issues on the chapters page
There's a handful of pretty minor layout issues on the chapters page ([Figma](https://www.figma.com/file/LqjK3Tpvd9KJ4buFArCJBQ/Saving-Satoshi?node-id=975%3A12149&t=nxCE26gS2fD24CVh-1)). See them in the image below.

image

### PR DESCRIPTION
**Pull request by vinayaksh42**
## Pull Request checklist

Please check if your PR fulfills the following requirements:

- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been reviewed and added / updated if needed (for bug fixes / features)
- [x] Build was run locally and any changes were pushed
- [x] Lint has passed locally and any fixes were made for failures

## Pull Request type
Please check the type of change your PR introduces:

- [x] :bug: Bugfix
- [ ] :sparkles: Feature
- [ ] :lipstick: :art: Code style update (formatting, renaming)
- [ ] :recycle: Refactoring (no functional changes, no api changes)
- [ ] :zap: Performance
- [ ] :pencil: Documentation content changes
- [ ] :wrench: Other (please describe):

## What is the current behavior?
desktop view:
Screenshot 2023-01-15 at 2 02 09 AM

mobile view:
Screenshot 2023-01-15 at 2 00 27 AM

## What is the new behavior?
desktop view:
Screenshot 2023-01-15 at 2 02 18 AM

mobile view:
Screenshot 2023-01-15 at 2 00 36 AM

## Close issue

resolved #63 

## Does this introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -6781,4 +6781,4 @@ zod@^3.19.1:
 zwitch@^2.0.0:
   version "2.0.4"
   resolved "https://registry.yarnpkg.com/zwitch/-/zwitch-2.0.4.tgz#c827d4b0acb76fc3e685a4c6ec2902d51070e9d7"
-  integrity sha512-bXE4cR/kVZhKZX/RjPEflHaKVhUVl85noU3v6b8apfQEc1x4A+zBxjZ4lN8LqGd6WZ3dl98pY4o717VFmoPp+A==
+  integrity sha512-bXE4cR/kVZhKZX/RjPEflHaKVhUVl85noU3v6b8apfQEc1x4A+zBxjZ4lN8LqGd6WZ3dl98pY4o717VFmoPp+A==
```
Comment (aureleoules) [inline yarn.lock:6784]: No need to push this

**Review by aureleoules — APPROVED** [review]
Great this looks good.

File: yarn.lock (hunk 1 above)
Comment (vinayaksh42) [reply yarn.lock]: I have removed the above mentioned changes from my pull request

**Review by GBKS — APPROVED** [review]
@vinayaksh42 thank your for your first contribution, this looks great.

There is still a layout jump in medium-width screens when toggling between info and challenge tabs on the Chapters page. I think the only way we can address this is with a containing element that is set to the taller height of both tab contents. That means we also render both and just show/hide them, so we can detect the height. A bit complicated, but not sure there's another way. Let's get your PR merged and tackle this separately.

**Pull request merged**